scaladoc/src/dotty/tools/scaladoc/site/SidebarParser.scala

```diff
@@ -51,6 +53,7 @@ object Sidebar:
       |  subsection: # optional - If not provided, pages are loaded from the index directory
       |    - <subsection> | <page>
       |  # either index or subsection needs to be present
+      |  # the only exception is subsection representing Blog which needs title == "Blog" and directory
```

Comment:
What do you think of relaxing the requirement of providing a `directory` (and using `blog` as a default value)?

What happens if someone creates a directory `_blog` but no entry `title: Blog` in the sidebar? Conversely, is it possible to create an entry with `title: Blog` and `index: _news/index.html`, for instance (to read the blog posts from the `_news` directory)?

Reply:
There I also have some doubts. It's nice to be able to map blog directory but on the other hand it creates additional complexity to configuration. My idea was to generate blog regardless of presence Blog subsection in YAML (The only condition would be existence of "_blog" directory) but it might be misleading since we use YAML to define which static sites appear. I see two solutions here:
 - We remove changes that I've made here and only change blog directory to "blog"
 - We handle Blog as a special subsection where we can provide output directory for blog (and maybe the input directory?) but I feel like Blog is a special element in static site and it's not trivial to implement this solution without increasing complexity of logic.

Reply:
I think it’s good to allow the users to define the blog entry in their sidebar so that they can control where it appears in the navigation menu. Yes, the blog is a special entry. Maybe it requires a special AST node as well to make this clearer in the logic?

Reply:
Do you think we should control whether blog should be rendered with yaml config?

Reply:
I think for now we should just repair dotty.epfl.ch (implement the second solution) and create an issue about customizing Blog with YAML.